Source/monster.cpp

```diff
@@ -3899,7 +3886,7 @@ void SetMapMonsters(const uint16_t *dunData, Point startPosition)
 			auto monsterId = static_cast<uint8_t>(SDL_SwapLE16(monsterLayer[j * width + i]));
 			if (monsterId != 0) {
 				int mtype = AddMonsterType(MonstConvTbl[monsterId - 1], PLACE_SPECIAL);
-				PlaceMonster(ActiveMonsterCount++, mtype, i + startPosition.x + 16, j + startPosition.y + 16);
+				PlaceMonster(ActiveMonsterCount++, mtype, startPosition + Displacement { i + 16, j + 16 });

```

Comment:
Pretty sure you've made all callers of this function pass in world coordinates, the border offset has already been added
```suggestion
				PlaceMonster(ActiveMonsterCount++, mtype, startPosition + Displacement { i, j });
```

Reply:
Good catch